spec/rubocop/cop/rspec/match_array_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::MatchArray do
+  it 'flags `match_array` with array literal arguments' do
+    expect_offense(<<-RUBY)
+      it { is_expected.to match_array([content1, content2]) }
```

Comment:
again, how about match_array([*content, content2])
Whilw I do not insist of support, I just want to make sure it doesn't break